src/Fixer/Whitespace/NoExtraConsecutiveBlankLinesFixer.php

```diff
@@ -91,7 +91,13 @@ public function configure(array $configuration = null)
                     $this->tokenKindCallbackMap[CT_USE_TRAIT] = 'removeBetweenUse';
                     break;
                 case 'curly_brace_open':
-                    $this->tokenEqualsMap['{'] = 'fixAfterToken';
+                    $this->tokenEqualsMap['{'] = 'fixAfterTokenIfMultiLineBody'; // i.e. not: CT_ARRAY_INDEX_CURLY_BRACE_OPEN
+                    break;
+                case 'square_brace_open':
+                    $this->tokenKindCallbackMap[CT_ARRAY_SQUARE_BRACE_OPEN] = 'fixAfterTokenIfMultiLineBody'; // typeless '[' tokens should not be fixed (to rare)
```

Comment:
confirmed, 
```php
<?php
array(

1
);`
```

```php
<?php
array(
1
);`
```

with this fixer and `parenthesis_brace_open`
I'll commit a test for this soon